docs/resource-sharing/os-backfill-containers.md

```diff
@@ -94,6 +95,15 @@ but it has other advantages such as supporting automounting of repositories.
 
 ### Adding CVMFS using cvmfsexec
 
+!!! important "EL7 hosts require an updated kernel"
+    EL7 hosts must have kernel version >= 3.10.0-1127 (run `uname -vr` to check) with user namespaces enabled and
+    network namespaces disabled.
```

Comment:
Actually, the network namespaces recommendation applies to EL8 as well.  That's not captured here.  Saying that network namespaces "must" be disabled is stronger than I would put it, but you can do that if you want to.

Reply:
Yeah, I think in this case I'd like to take a stronger stance